### ISSUE TITLE
git fails for some repos on last06e

### ISSUE THREAD

**Issue by EastEriq** [blumzi/LAST_tools#2]
note the [FAIL]s. I think it is a credential problem, not all repos are happy with the token provided? We should check also the source repos, maybe it is them, missing an authorization for Cecco di Ronchitti.

```
ocs@last06e:~$ sudo last-tool --nodeps enforce last-software
[sudo] password for ocs: 

[SECT] Last-software
[INFO] Fetching the LAST software from github ...
[INFO] Pulling from https://github.com/EranOfek/AstroPack.git into "/home/ocs/matlab/AstroPack" ...
remote: Enumerating objects: 77, done.
remote: Counting objects: 100% (73/73), done.
remote: Compressing objects: 100% (23/23), done.
remote: Total 77 (delta 54), reused 67 (delta 49), pack-reused 4
Unpacking objects: 100% (77/77), 17.13 KiB | 701.00 KiB/s, done.
From https://github.com/EranOfek/AstroPack
   b9feed3a..63d1419a  dev1          -> origin/dev1
 * [new branch]        chen_planner1 -> origin/chen_planner1
[ OK ] Pulled from "https://github.com/EranOfek/AstroPack.git" into "/home/ocs/matlab/AstroPack".
[INFO] Pulling from https://github.com/EastEriq/LAST_AtikCamera.git into "/home/ocs/matlab/LAST/LAST_AtikCamera" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_AtikCamera.git" into "/home/ocs/matlab/LAST/LAST_AtikCamera".
[INFO] Pulling from https://github.com/EastEriq/LAST_CelestronFocusMotor.git into "/home/ocs/matlab/LAST/LAST_CelestronFocusMotor" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_CelestronFocusMotor.git" into "/home/ocs/matlab/LAST/LAST_CelestronFocusMotor".
[INFO] Pulling from https://github.com/EranOfek/LAST_config.git into "/home/ocs/matlab/LAST/LAST_config" ...
[ OK ] Pulled from "https://github.com/EranOfek/LAST_config.git" into "/home/ocs/matlab/LAST/LAST_config".
[INFO] Pulling from https://github.com/EastEriq/LAST_EnclosureControl into "/home/ocs/matlab/LAST/LAST_EnclosureControl" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_EnclosureControl" into "/home/ocs/matlab/LAST/LAST_EnclosureControl".
[INFO] Pulling from https://github.com/EastEriq/LAST_EnvironmentArduinoSensors.git into "/home/ocs/matlab/LAST/LAST_EnvironmentArduinoSensors" ...
fatal: could not read Username for 'https://github.com': No such device or address
error: Could not fetch origin
[FAIL] Could not pull from "https://github.com/EastEriq/LAST_EnvironmentArduinoSensors.git" into "/home/ocs/matlab/LAST/LAST_EnvironmentArduinoSensors" (status: 1)
[INFO] Pulling from https://github.com/EastEriq/LAST_Handle.git into "/home/ocs/matlab/LAST/LAST_Handle" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_Handle.git" into "/home/ocs/matlab/LAST/LAST_Handle".
[INFO] Pulling from https://github.com/EastEriq/LAST_iOptronCEM120.git into "/home/ocs/matlab/LAST/LAST_iOptronCEM120" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_iOptronCEM120.git" into "/home/ocs/matlab/LAST/LAST_iOptronCEM120".
[INFO] Pulling from https://github.com/EastEriq/LAST_Messaging.git into "/home/ocs/matlab/LAST/LAST_Messaging" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_Messaging.git" into "/home/ocs/matlab/LAST/LAST_Messaging".
[INFO] Pulling from https://github.com/EastEriq/LAST_MountArduinoSensors.git into "/home/ocs/matlab/LAST/LAST_MountArduinoSensors" ...
fatal: could not read Username for 'https://github.com': No such device or address
error: Could not fetch origin
[FAIL] Could not pull from "https://github.com/EastEriq/LAST_MountArduinoSensors.git" into "/home/ocs/matlab/LAST/LAST_MountArduinoSensors" (status: 1)
[INFO] Pulling from https://github.com/PolishookDavid/LAST_OCS.git into "/home/ocs/matlab/LAST/LAST_OCS" ...
fatal: could not read Username for 'https://github.com': No such device or address
error: Could not fetch origin
[FAIL] Could not pull from "https://github.com/PolishookDavid/LAST_OCS.git" into "/home/ocs/matlab/LAST/LAST_OCS" (status: 1)
[INFO] Pulling from https://github.com/EastEriq/LAST_QHYccd.git into "/home/ocs/matlab/LAST/LAST_QHYccd" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_QHYccd.git" into "/home/ocs/matlab/LAST/LAST_QHYccd".
[INFO] Pulling from https://github.com/EastEriq/LAST_tinycontrolIPpowerSocket.git into "/home/ocs/matlab/LAST/LAST_tinycontrolIPpowerSocket" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_tinycontrolIPpowerSocket.git" into "/home/ocs/matlab/LAST/LAST_tinycontrolIPpowerSocket".
[INFO] Pulling from https://github.com/blumzi/LAST_tools into "/home/ocs/matlab/LAST/LAST_tools" ...
[ OK ] Pulled from "https://github.com/blumzi/LAST_tools" into "/home/ocs/matlab/LAST/LAST_tools".
[INFO] Pulling from https://github.com/EastEriq/LAST_USBserial.git into "/home/ocs/matlab/LAST/LAST_USBserial" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_USBserial.git" into "/home/ocs/matlab/LAST/LAST_USBserial".
[INFO] Pulling from https://github.com/EastEriq/LAST_XerxesMount.git into "/home/ocs/matlab/LAST/LAST_XerxesMount" ...
fatal: could not read Username for 'https://github.com': No such device or address
error: Could not fetch origin
[FAIL] Could not pull from "https://github.com/EastEriq/LAST_XerxesMount.git" into "/home/ocs/matlab/LAST/LAST_XerxesMount" (status: 1)
[INFO] Pulling from https://github.com/EastEriq/LAST_ZWOASICamera.git into "/home/ocs/matlab/LAST/LAST_ZWOASICamera" ...
[ OK ] Pulled from "https://github.com/EastEriq/LAST_ZWOASICamera.git" into "/home/ocs/matlab/LAST/LAST_ZWOASICamera".
[WARN] No LAST-CONTAINER.  Only the github repositories were enforced
```

**Comment by EastEriq** [comment]
Repos are:
Mine:
- LAST_EnvironmentArduinoSensors
- LAST_MountArduinoSensors
- LAST_XerxesMount
- 
David's
- LAST_OCS

**Comment by blumzi** [comment]
Fixed w/ commit c7924c1.  Added config of remote.origin.url before pull
